src/sync_protocol/include/persistent_queue_storage.hpp

```diff
@@ -0,0 +1,63 @@
+/* Copyright (C) 2015, Wazuh Inc.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#pragma once
+
+#include "ipersistent_queue_storage.hpp"
+#include "../../shared_modules/utils/sqlite3Wrapper.hpp"
```

Comment:
Can this be solved in the CMakeLists.txt?

Reply:
Done!